app/keepers/keepers.go

```diff
@@ -303,6 +322,30 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 		authtypes.NewModuleAddress(govtypes.ModuleName).String(),
 	)
 
+	appKeepers.keys[ibchookstypes.StoreKey] = storetypes.NewKVStoreKey(ibchookstypes.StoreKey)
```

Comment:
Not here. 
Supposed to go in : `func KVStoreKeys() []string`

Reply:
As we use GenerateKeys() in special keepers

Reply:
Already in thre. Please remove.

Reply:
this was during initial compilation testing removed